src/lib/buzz_os.zig

```diff
@@ -133,6 +133,8 @@ fn handleSpawnError(ctx: *api.NativeCtx, err: anytype) void {
 
         error.OutOfMemory => @panic("Out of memory"),
         error.Unexpected => ctx.vm.pushError("errors.UnexpectedError", null),
+
+        else => ctx.vm.pushError("errors.UnexpectedError", null),
```

Comment:
You should not need to do this. If its a new error, add it to lib/error.buzz

Reply:
added default error for the switch handle